pkg/haproxy/haproxy.go

```diff
@@ -300,6 +321,8 @@ func haproxyContainer(cr *apiv1alpha1.PerconaServerMySQL) corev1.Container {
 		EnvFrom:         spec.EnvFrom,
 		Command:         []string{"/opt/percona/haproxy-entrypoint.sh"},
 		Args:            []string{"haproxy"},
+		ReadinessProbe:  &readinessProbe,
+		LivenessProbe:   &livenessProbe,
```

Comment:
Should we have a version check for adding these? If I'm not mistaken, we discussed that at some point that we should for ps operator.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/975/commits/446729add0f51a0eb3af36545989734098e4d6d0